analysis/.gitignore

```diff
@@ -0,0 +1,248 @@
+
```

Comment:
why have 2 git ignore??

Reply:
sorry it was a my mistake
I removed one in delphys/analysis